### DISCUSSION TITLE
No response from GitHub Support after waiting 90 days

### DISCUSSION THREAD

**Discussion by arcanistzed** [community/community#81328] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have a GitHub Support request that has received no response after waiting over 90 days. I made other requests since I thought the first one might have gotten lost. None of these requests have received responses.
Is there any way that I can contact GitHub directly?

**Comment by shanestover8991** [comment]
I recommend StackOverflow.com or reddit.com/r/ExperiencedDevelopers. Good luck! Happy December

**Reply by arcanistzed** [reply]
Thank you for the advice. My request is for GitHub Teams for Nonprofits, not a technical question, and I believe that only GitHub Support staff can assist with this.

**Reply by shanestover8991** [reply]
Cool! There's a cool magazine to read called Nonprofit Quarterly or w/e else while you wait for their timely response if you like, and a nonprofit I find inspiring is called Remote Shoals. Enjoy you Thursday!
